specs/SetMemoryUsageLevel.md

```diff
@@ -0,0 +1,160 @@
+# Background
+Some WebView2 apps want to continue to run scripts while inactive and therefore cannot make usage of `TrySuspend` and `Resume` APIs to reduce resource consumption.
+We are introducing WebView2 API to reduce memory usage only for this type of inactive WebViews.
+
+# Description
+You may set the `MemoryUsageTargetLevel` property to have the WebView2 consume less memory and going back to normal usage. This is useful when your Win32 app becomes invisible, 
+but still wants to have script running or monitoring requests from network.
+
```

Comment:
It took me some reading to understand what "target" means here. I think it means that the WebView will try to limit its memory usage, but the enum is already vague enough (`Normal` and `Low`).

Suggest `MemoryUsageLimit` .

Reply:
"Limit" suggests enforcement a restriction and potentially enforcing it. That is not what we are doing.
We are trying to reduce memory consumption. How about webview.MemoryUsageProfile = MemoryUsageProfileLow?

Reply:
Thought about this more. MemoryUsageProfile is not good either. It is vague and might cause people to think about low end devices.
Open to better names. If no better suggestion, will use the current MemoryUsageTargetLevel.

Reply:
Will use the current MemoryUsageTargetLevel name.